source/common/upstream/upstream_impl.cc

```diff
@@ -1472,7 +1472,9 @@ void PriorityStateManager::updateClusterPrioritySet(
   std::vector<HostVector> per_locality;
 
   // If we are configured for locality weighted LB we populate the locality weights.
```

Comment:
Maybe update this comment with the reasoning for including this for lb policies? My understand is that the extension *may* want to use this, so we include this just in case. Maybe even a TODO to have this happen only if the extension requests it somehow

Reply:
Done and done. Your understanding matches mine :-)